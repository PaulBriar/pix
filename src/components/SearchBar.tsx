import React from 'react';

class SearchBar extends React.Component<any, any> {
    public render() {
        return (
            <div className="ui segment">
                <form className="ui form">
                    <div className="field">
                        <input type="text" placeholder="Image Search"/>
                    </div>
                </form>
            </div>
        )
    }
};

export default SearchBar;