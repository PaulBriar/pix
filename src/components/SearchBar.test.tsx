import React from 'react';
import { shallow } from 'enzyme';

import SearchBar from './SearchBar';

describe('SearchBar', () => {
    it('Should render the SearchBar component', () => {
        const component = shallow(<SearchBar/>);
        expect(component).toMatchSnapshot();
    });
});