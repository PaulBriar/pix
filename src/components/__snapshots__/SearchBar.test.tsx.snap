// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar Should render the SearchBar component 1`] = `
<div
  className="ui segment"
>
  <form
    className="ui form"
  >
    <div
      className="field"
    >
      <input
        placeholder="Image Search"
        type="text"
      />
    </div>
  </form>
</div>
`;
